### ISSUE TITLE
Training strategy

### ISSUE THREAD

**Issue by liu15509348793** [ultralytics/ultralytics#16891]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I want to use the data of the train and val parts to train together, and use the val part to verify the progress, how should I set it? I need to modify the parameters of yaml or train, I am using the yolov11obb model, and my yaml file is as follows:
path: datasets/DOTA1.0_YOLO_Split # dataset root dir
train: images/train
val: images/val
test: images/test

nc: 15
names:
  0: plane
  1: baseball-diamond
  2: bridge
  3: ground-track-field
  4: small-vehicle
  5: large-vehicle
  6: ship
  7: tennis-court
  8: basketball-court
  9: storage-tank
  10: soccer-ball-field
  11: roundabout
  12: harbor
  13: swimming-pool
  14: helicopter

**Comment by UltralyticsAssistant** [comment]
👋 Hello @liu15509348793, thank you for reaching out to Ultralytics 🚀! This is an automated response, and an Ultralytics engineer will assist you soon.

We suggest checking our [Docs](https://docs.ultralytics.com) for guidance on training strategies. For your setup, ensure you explore [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to optimize your configuration when using the yolov11obb model.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to assist us in debugging.

To connect with others or get real-time help, join our [Discord](https://ultralytics.com/discord) 🎧. For deeper discussions, our [Discourse](https://community.ultralytics.com) or [Subreddit](https://reddit.com/r/ultralytics) are great places to explore.

## Upgrade

Ensure you're using the latest version by upgrading the `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in these up-to-date verified environments:

- **Notebooks** with free GPU:
  - <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>
  - <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
  - <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing, indicating correct operation across various platforms.

**Comment by Y-T-G** [comment]
You can add multiple train folders
```
train: 
- images/train
- images/val
```

The rest is the same.

**Comment by liu15509348793** [comment]
> You can add multiple train folders
> 
> ```
> train: 
> - images/train
> - images/val
> ```
> 
> The rest is the same.

Thank you for your reply, and I would like to ask about the weight file obtained by the official, such as how the yolo11s-obb.pt is trained, what is the specific cutting method of the DOTA dataset, whether the data training using only train or train and val is used, how many epochs have been trained in total, and parameters such as batchsize, imgsz, etc., can you help me answer

**Comment by Y-T-G** [comment]
It was trained in `train`. You can find the training arguments from the model.

```python
from ultralytics import YOLO

model = YOLO("yolo11n.pt")
print(model.ckpt["train_args"]
```

**Comment by liu15509348793** [comment]
> It was trained in `train`. You can find the training arguments from the model.
> 
> ```python
> from ultralytics import YOLO
> 
> model = YOLO("yolo11s-obb.pt")
> print(model.ckpt["train_args"]
> ```

Thanks for the reply, I did find the training parameters your way, but I still don't know how the DOTA dataset for training is cut and whether the training was done with only train data or with train and val together

**Comment by Y-T-G** [comment]
Just `train`.

**Comment by liu15509348793** [comment]
> Just `train`.
Is this how the split parameters of the DOTA dataset are set?

from ultralytics.data.split_dota import split_test, split_trainval

# split train and val set, with labels.
split_trainval(
    data_root="path/to/DOTAv1.0/",
    save_dir="path/to/DOTAv1.0-split/",
    rates=[0.5, 1.0, 1.5],  # multiscale
    gap=500,
)
# split test set, without labels.
split_test(
    data_root="path/to/DOTAv1.0/",
    save_dir="path/to/DOTAv1.0-split/",
    rates=[0.5, 1.0, 1.5],  # multiscale
    gap=500,
)

**Comment by glenn-jocher** [comment]
@liu15509348793 yes, those parameters are used for splitting the DOTA dataset.

**Comment by liu15509348793** [comment]
> @liu15509348793 yes, those parameters are used for splitting the DOTA dataset.

Are the values of rate and gap correct? What are the pre-trained weights you get for this segmented dataset? I see a lot of people setting rate=1.0 and gap=200

**Comment by glenn-jocher** [comment]
The values for `rate` and `gap` can vary based on your specific needs. The pre-trained weights were obtained using the default settings, which may differ from what others use. Adjust these parameters to suit your dataset requirements.
